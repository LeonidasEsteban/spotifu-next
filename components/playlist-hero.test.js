import React from 'react'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import PlaylistHero from './playlist-hero.js'

const props = {
  name: 'Playlist de leo',
  description: 'Esta es un playlist bonita',
  type: 'show',
  image: ''
}

test('PlaylistHero',  () => {
  render(<PlaylistHero {...props}/>)
  expect(screen.getByText(/Playlist de leo/i)).toBeInTheDocument()
})

test('snapshot PlaylistHero', () => {
  const { asFragment } = render(<PlaylistHero {...props} />)
  expect(asFragment()).toMatchSnapshot()
})

