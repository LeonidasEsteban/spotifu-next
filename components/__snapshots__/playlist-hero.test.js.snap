// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot PlaylistHero 1`] = `
<DocumentFragment>
  <div
    class="playlistHero"
  >
    <img
      alt=""
      height="232"
      src=""
      width="232"
    />
    <div
      class="playlistHero-content"
    >
      <p
        class="playlistHero-type"
      >
        show
      </p>
      <h1
        class="playlistHero-name"
      >
        Playlist de leo
      </h1>
      <h2
        class="playlistHero-description"
      >
        Esta es un playlist bonita
      </h2>
    </div>
  </div>
</DocumentFragment>
`;
